README.md

```diff
@@ -30,12 +30,13 @@ To setup your Docker instance simply follow these steps:
 3. Configure the environment variables as described in the [CONFIGURE.md](CONFIGURE.md)
 4. Run the container
 5. Setup a crontab outside the container to run `* * * * * docker exec --user www-data -i control-center php artisan schedule:run >/dev/null` every minute. This patches into the container and runs the required cronjobs.
-6. Bind the 8080 (HTTP) and/or 8443 (HTTPS) port to your reverse proxy or similar.
+6. To ensure that users will not need to login after each time you re-deploy/upgrade the container, enter the container and run `php artisan key:get`. Copy the key and set it as the `APP_KEY` environment variable in your docker configuration.
```

Comment:
quick suggestion: WDYT about having a line with `docker exec` that does the `php artisan key:get`? I can chuck in a suggestion later.